app/models/shipit/deploy_spec/lerna_discovery.rb

```diff
@@ -83,15 +83,16 @@ def publish_independent_packages
 
       def publish_fixed_version_packages
         check_tags = 'assert-lerna-fixed-version-tag'
+        version = lerna_version
```

Comment:
Isn't this always going to be `fixed`?

Reply:
Oh, this is fixed version logic, eh.  Nevermind 😄

Reply:
@GoodForOneFare with fixed versioning the `version` key is the actual current version for all packages (e.g. `"version": "1.3.1-alpha.2"`)